### PR TITLE
docs: fix workspaces example

### PR DESCRIPTION
**Pull request by fvilers**
Specify the right path for workspace "a"

### CODE REVIEW

File: .github/workflows/ci-release.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ on:
       ref:
         required: true
         type: string
-        default: latest
```
Comment (wraithgar) [inline .github/workflows/ci-release.yml]: Looks like a commit from another branch snuck in here. You'll probably want to start w/ a fresh branch off of `latest`, or rebase this, or drop this commit from your branch.

**Comment by infamous100** [comment]
Did not authorize trade

**Comment by fvilers** [comment]
Hello, I dropped the commit we didn't want. Hope this helps.

**Review by wraithgar — DISMISSED** [review]

File: docs/lib/content/using-npm/workspaces.md [hunk 2]
```diff
@@ -102,7 +102,7 @@ the npm installer that package should be added as a dependency of the provided
 workspace:
 
 ```
-npm install abbrev -w a
+npm install abbrev -w packages/a
```
Comment (wraithgar) [inline docs/lib/content/using-npm/workspaces.md:105]: This was actually correct, and included to show that the `-w` flag accepts the name of the workspace, not just the path.

Comment (fvilers) [reply docs/lib/content/using-npm/workspaces.md:105]: It's not working here (Windows 10, Node 18.15.0)

image

Comment (wraithgar) [reply docs/lib/content/using-npm/workspaces.md:105]: You want `npm install abbrev -w my-budget`

Comment (wraithgar) [reply docs/lib/content/using-npm/workspaces.md:105]: What is the `name` attribute of the `package.json` in `packages/app`?

Comment (fvilers) [reply docs/lib/content/using-npm/workspaces.md:105]: OK, I get it. The argument to `-w` is not the directory name but the package name defined in `package.json`. Sorry for the misunderstanding, but it's not precisely stated in the documentation. The example used in documentation is directory structure oriented, hence my confusion.

Comment (wraithgar) [reply docs/lib/content/using-npm/workspaces.md:105]: Yes!  Perhaps the examples could be updated so the directory and package names differ, to make this more clear?

**Pull request closed without merging**